GuardiansoftheSpectrum/app/src/main/java/edu/colorado/gots/guardiansofthespectrum/BaseActivity.java

```diff
@@ -55,6 +57,10 @@ public boolean onCreateOptionsMenu(Menu menu) {
         MenuInflater mMenuInflater = getMenuInflater();
         mMenuInflater.inflate(R.menu.our_menu, menu);
 
+        Toolbar mToolbar = (Toolbar) findViewById(R.id.toolbar);
```

Comment:
These lines will have to be removed. Otherwise, we will continually call this function in a loop because the toolbar options menu will keep being set. You can see this play out in the debug messages on the Galaxy S7 phone.